### PR TITLE
Add material on adaptive monitoring

### PR DESCRIPTION
**Pull request by SpencerDawkins**

### CODE REVIEW

**Comment by SpencerDawkins** [comment]
@gitnnelg, please let me know if this is the kind of thing you were asking for. Thanks!

**Comment by gitnnelg** [comment]
It's a very good step in the right direction. Basically very close to resolved.     I've made specific comments/suggestion in the PR.

File: draft-ietf-mops-streaming-opcons.md [hunk 1]
```diff
@@ -375,22 +380,28 @@ then chooses a bitrate for each of the video, audio, subtitles and metadata (amo
...
 
 ABR playback is commonly implemented by streaming clients using HLS
 {{RFC8216}} or DASH {{DASH}} to perform a reliable segmented delivery
-of media over HTTP. Different implementations use different strategies {{ABRSurvey}}, often proprietary
+of media over HTTP. Different implementations use different strategies {{ABRSurvey}}, often relying on proprietary
 algorithms (called rate adaptation or bitrate selection algorithms) to perform available
-bandwidth estimation/prediction and the bitrate selection. Most clients only use passive observations, i.e., they do not generate probe traffic to measure the available bandwidth. 
+bandwidth estimation/prediction and the bitrate selection. 
+
+Most clients only use passive observations, i.e., they do not generate probe traffic to measure the available bandwidth.
```
Comment (gitnnelg) [inline draft-ietf-mops-streaming-opcons.md:395]: Many delivery systems do a test delivery to clients to do an initial test of bandwidth for the playout to follow.    So perhaps a better wording would be:

Many server-player systems will do an initial probe or a very simple throughput speed test at the start of a video play.   This is done to get a rough sense of the highest video bit rate in the ABR ladder that the network between the server and player will likely be expected under current network conditions.   After the initial testing, clients tend to rely upon passive network observations and will make use of player side statistics such buffer fill rates to monitor and respond to changing network conditions.

File: draft-ietf-mops-streaming-opcons.md [hunk 2]
```diff
@@ -375,22 +380,28 @@ then chooses a bitrate for each of the video, audio, subtitles and metadata (amo
...
 the network or changes in capabilities occur during the playback (such as available memory, CPU, display size, etc.).
 
-The choice of bitrate occurs within the context of optimizing for
-some metric monitored by the client, such as highest achievable
-video quality or lowest chances for a rebuffering (playback stall).
+##Adaptive Encoding {#adapt-encode}
 
-##Segmented Delivery
+Media servers can provide media streams at various bitrates because the media has been encoded at various bitrates. This is a so-called "ladder" of bitrates, that can be offered to media players as part of the manifest that describes the media being requested by the media player, so that the media player can select among the available bitrate choices.
+
```
Comment (gitnnelg) [inline draft-ietf-mops-streaming-opcons.md:386]: The media server may also choose to alter which bitrates are made available to players by adding or removing bitrate options from the ladder delivered to the player in subsequent manifests built and sent to the player.   

In this way both the player, through its selection of bitrate to request from the manifest, and the server, through its construction of the bitrates offered in the manifest, are able to affect the network utilization.

**Comment by SpencerDawkins** [comment]
I merged this PR based on comments above with @gitnnelg so that we'll have it in the next draft we submit.

File: draft-ietf-mops-streaming-opcons.md (hunk 1 above)
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md]: I used this text with very few changes. Thanks for sending text!

File: draft-ietf-mops-streaming-opcons.md (hunk 2 above)
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:491]: I used this text with like, 2 editorial changes. Thanks for your help on this!

**Pull request merged**